cfme/tests/containers/test_containers_smartstate_analysis.py

```diff
@@ -79,6 +104,15 @@ def test_containers_smartstate_analysis(provider, test_item, soft_assert):
         if not soft_assert(hasattr(table, attr),
                 '{} table has missing attribute \'{}\''.format(tbl, attr)):
             continue
-        value = getattr(table, attr).value
+        wait_for_retval = wait_for(lambda: getattr(table, attr).value,
+                                   message='Trying to get attribute "{}" of table "{}"'.format(
+                                       attr, tbl),
+                                   fail_func=image_obj.summary.reload,
+                                   delay=5, num_sec=120, silent_failure=True)
```

Comment:
Because I would like it to continue to the next checks and not just fail here. see the next ~4 lines (soft_assert)